next.clayui.com/content/docs/components/drop-down.mdx

```diff
@@ -36,16 +39,81 @@ import {DropDown} from '../../../src/components/clay/DropDown';
 	Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/dropdowns/">Dropdown Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
 </div>
 
-DropDown is delivered for composition offering flexibility for customization.
+<div class="clay-site-alert alert alert-warning">
+	<strong class="lead">Warning</strong>
+	Before you get started with this doc, read about <a href="/docs/foundations/composing.html" target="_blank">Clay's fundamentals of composition</a> to move on. The information below is assuming you have read about <a href="/docs/foundations/composing.html" target="_blank">Clay's compositional philosophy</a> and learned about the terms.
+</div>
+
+DropDown is delivered for composition offering flexibility for customization. DropDown's composition gives you the space to create more complex DropDowns, such as adding a form within DropDown, and creating fast DropDowns with the high-level `ClayDropDownWithItems` component that covers most Lexicon specification use cases.
 
 <DropDown />
 
+## Composing
+
 You may want to use only the content of DropDown for other cases, such as mounting an `Autocomplete`, so use `DropDown.Menu` instead of `DropDown` which gives you the flexibility to only control content without a trigger.
 
 <div class="clay-site-alert alert alert-warning">
 	<code class="language-text">DropDown</code> consists of a <code class="language-text">DropDown.Menu</code> that controls the trigger and uses it as the basis for aligning DropDown content.
 </div>
 
+#### Controlling Trigger
```

Comment:
I thought of putting a name other than the component just to get the ToC viewer's attention, `ClayDropDown.Menu` might not mean much to some.

Reply:
I think the component name is helpful, but we could also call it something like "Using dropdown without a trigger." Mostly because I think "Controlling Trigger" isn't really helpful or descriptive what it is actually talking about.

Reply:
"Using dropdown without a trigger" looks better, I'll go with him.